csharp/ql/test/library-tests/delegates/delegates.cs

```diff
@@ -83,4 +83,17 @@ static void Main()
 
     }
 
+    class E
+    {
+        Action<int> Field;
+        Action<int> Property { get; set; }
+
+        void M()
+        {
+            this.Field(0);
```

Comment:
Do we have the same problem with function pointers? (`unsafe delegate*<int, void> Field2;`)

Reply:
We did, yes, but that is also fixed by this PR. I have added additional tests to witness.